<launch>
<arg name="robot" default="rossumo1" />
<arg name="device" default="/dev/input/js1" />
<group ns="$(arg robot)">
  <!-- Driver for the joystick. -->
  <node pkg="joy" type="joy_node" name="joy" output="screen" >
    <param name="dev" value="$(arg device)" />
    <param name="autorepeat_rate" value="25" />
  </node>

  <!-- Sumo specific teleoperation driver -->
  <param name="axis_180_360turn" value="5"/>
  <param name="axis_90turn" value="4"/>
  <param name="axis_angular" value="0"/>
  <param name="axis_linear" value="3"/>
  <param name="button_anim" value="6"/>
  <param name="button_high_jump" value="4"/>
  <param name="button_posture" value="5"/>

  <param name="scale_angular" value="1"/>
  <param name="scale_linear" value="1"/>

  <node pkg="rossumo" type="sumo_teleop_joy.exe" name="teleop" output="screen">
    <remap from="cmd_vel" to="cmd_vel_norm"/>
  </node>
</group>
</launch>
